samba/data/public/index.html

```diff
@@ -0,0 +1,333 @@
+
+<body>
+    <section class="container app-container">
+        <div class="app">
+            <img class="app-icon" src="samba.svg" width="256" height="256" />
+            <div class="app-details">
+                <span class="app-status success">&#9679; Running</span>
+                <h1 class="app-name">Samba</h1>
+            </div>
+        </div>
+        <p class="text-muted">
+            <hr />
+        <h2>How to Connect to the Umbrel SMB Server</h2>
+        <p>Follow these steps to connect to the Umbrel SMB Server:</p>
+
+        <h3>For Windows Users</h3>
+        <ol>
+            <li>Open <strong>File Explorer</strong>.</li>
+            <li>In the address bar, type <code>\\umbrel.local</code> and press <strong>Enter</strong>.</li>
+            <li>When prompted, enter the your credentials (check Umbrel dashboard -> Samba -> Right-Click -> Show
+                default
+                credentials).
+            </li>
+            <li>You should now see all of your umbrelOS data like downloads, movies, music, etc...</li>
+        </ol>
+
+        <h3>For macOS Users</h3>
+        <ol>
+            <li>Open <strong>Finder</strong>.</li>
+            <li>In the menu bar, click on <strong>Go</strong> and then select <strong>Connect to Server...</strong>.
+            </li>
+            <li>In the Server Address field, type <code>smb://umbrel.local</code> and click
+                <strong>Connect</strong>.
+            </li>
+            <li>When prompted, enter the your credentials (check Umbrel dashboard -> Samba -> Right-Click -> Show
+                default
+                credentials).
+            </li>
+            <li>You should now see all of your umbrelOS data like downloads, movies, music, etc...</li>
+        </ol>
+
+        <h3>For Linux Users</h3>
+        <ol>
+            <li>Open a file manager (such as <strong>Nautilus</strong> or <strong>Dolphin</strong>).</li>
+            <li>In the address bar, type <code>smb://umbrel.local</code> and press <strong>Enter</strong>.</li>
+            <li>When prompted, enter the your credentials (check Umbrel dashboard -> Samba -> Right-Click -> Show
+                default
+                credentials).
+            </li>
+            <li>You should now see all of your umbrelOS data like downloads, movies, music, etc...</li>
+        </ol>
+
+        <p>If you encounter any issues, make sure that the Umbrel SMB Server is running and that you have the
+            correct
+            permissions to access it. Additionally, ensure that your firewall or antivirus software is not blocking
+            the
+            connection.</p>
+
+        <p>For more detailed configuration information, <a href="smb.conf">click here to check your config file</a>.
+            You can change the config by opening the terminal (Settings -> Advanced Settings -> Terminal) and manually
+            modifying the smb file (e.g. with nano umbrel/app-data/samba/data/smb.conf).
+        </p>
```

Comment:
If we decide to leave out the `smb.conf` altogether, we should change this description and tell users to edit the `docker-compose.yml` directly in order to add a custom `smb.conf` file.